webapp/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:16
```

Comment:
Is the reason for setting nodejs 16 or higher because of the npm version?

Reply:
Yes.

the current lockfile version is 2, and `npm` uses lockfile version 2 since `npm 7`, also `nodejs 15+` uses `npm 7`, so I chose 16, the most recent stable release.